cmd/rollkit/commands/run_node.go

```diff
@@ -58,7 +58,7 @@ func NewRunNodeCmd() *cobra.Command {
 				return err
 			}
 
-			// use aggregator by default
+			// use aggregator by default if the flag is not specified explicitly
 			if !cmd.Flags().Lookup("rollkit.aggregator").Changed {
```

Comment:
i don't think you need this code, now that by default if the flag is not passed, it is set to true.

Reply:
nevermind. you still need it.